lib/publiccloud/instance.pm

```diff
@@ -921,15 +921,26 @@ sub do_systemd_analyze_time {
 
 sub upload_supportconfig_log {
     my ($self, %args) = @_;
-    my $timeout = 600;
+    my $timeout = 600 + (is_sle('=12-SP5') ? 800 : 0);
     my $start = time();
     my $logs = "/var/tmp/scc_supportconfig";
+    # Eventual comma-separated tokens list to exclude
+    my $exclude = get_var('PUBLIC_CLOUD_SUPPORTCONFIG_EXCLUDE', 'AUDIT');
+    # To remove exclusions, _EXCLUDE='-'
+    $exclude = undef if ($exclude eq '-');
```

Comment:
Why use `-` and not simply the empty string for this?

```suggestion
    my $exclude = get_var('PUBLIC_CLOUD_SUPPORTCONFIG_EXCLUDE', 'AUDIT');
```

Reply:
I explained this already, in the discussion with Anton https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/23510#discussion_r2409588595, because he suggested to keep the 'AUDIT' default and this prevents to use the empty string, that will  trigger the default instead.